pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+For a PHEP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate PyHC projects unduly. Finally, a proposed enhancement must be "pythonic" in order to be accepted by the community. (However, "pythonic" is an imprecise term; it may be defined as whatever is acceptable to the community. This logic is intentionally circular.)
+
+The editors must post oronouncements of PHEP resolution to the PyHC mailing list.
+
+Once a PHEP has been accepted, the reference implementation, if applicable, must be completed. When the reference implementation is complete and incorporated into a PyHC project's source code repository, the status will be changed to "Final". PHEPs which do not require a reference implementation move immediately to "Final" upon acceptance.
+
+If a PHEP requires additional design and interface feedback before committing to long term stability, it must remain in "Draft". Alternatively, its scope can be reduced (e.g. by deferring some features to later PHEPs).
+
+If no progress is being made on the PHEP, the editor should consult the author. The author may resume work on the PHEP, [transfer it to another author](transferring-phep-ownership), or withdraw it. If the author takes none of these actions, the editor may choose to bring the PHEP to the community for approval or rejection in its current form.
+
+If a PHEP fails to find consensus, the author may choose to modify it for reconsideration in the future. The PHEP may also be "Withdrawn" if the PHEP author themself has decided that the PHEP is actually a bad idea, or has accepted that a competing proposal is a better alternative. A PHEP can also be "Rejected" if it fails to find consensus and the author is not willing or able to modify it. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+When a PHEP changes state, the PHEP text must be updated accordingly. In addition to updating the Status field, at the very least the Resolution header should be added with a direct link to the minutes of the telecons and meetings where a decision on the PHEP was made.
+
+Final PHEPs can also be replaced by a different PHEP, rendering the original obsolete.
+
+The possible paths of the status of PHEPs are as follows:
+
+![PHEP status flow diagram](phep-0001/process_flow.svg)
+
+While not shown in the diagram, "Accepted" PHEPs may technically move to "Rejected" or "Withdrawn" even after acceptance. This will only occur if the implementation process reveals fundamental flaws in the design that were not noticed prior to acceptance of the PHEP.
+
+Upon reaching "Final", "Withdrawn", or "Rejected" status, the editors update the preamble and merge the PR into `main`.
```

Comment:
Do we need/want to place restrictions on deleting the branch after merge? I don't know offhand what that does to the conversation...if GH keeps around the commits so that we can see what comments are tied to, this isn't a problem.

Reply:
It would be fine to delete a branch after merging since it's possible to restore it later.  (I've accidentally deleted branches enough to really appreciate this feature!) We could also make a point to not squash PRs when merging.

Reply:
+1 to deleting branches after merging.

Reply:
Marking this thread resolved without further edits, as I don't think we need to explicitly require one or the other (since the branch is on the submitter's fork, it's up to them).

I checked on some old SpacePy PR's and even after deleting the branch, if people commented on particular lines of the diff the diff and comments are still there.